### DISCUSSION TITLE
.zshrc:5: command too long  ^@^@^@

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [ohmyzsh/ohmyzsh#10073] [Help] [answered]
## How can i solve this problem ?

![Screen Shot 2021-07-25 at 2 31 15 PM](https://user-images.githubusercontent.com/44552983/126892903-92a438ea-da37-41cb-9a8a-fd728d465aaf.png)

**Comment by mcornella** [comment, marked as answer]
You have NULL characters in your .zshrc file. Edit it with a text editor to remove them.
